modules/api/README.md

```diff
@@ -0,0 +1,50 @@
+# API
+
+## Purpose
+
+The api module is intended to provide a stable, versioned REST interface
+for interacting with the core LORIS functionality without requiring backend
+SQL access.
+
+## Intended Users
+
+The module has 2 intended use cases:
+1. Users (or creators) of LORIS data that are external to the LORIS core
+   software that must extract or import data from LORIS.
+2. Internal LORIS modules which retrieve data from the API.
+
+## Scope
+
+The API only provides information about concepts that are cross-module
+or data that is of general concern to users. 
+
+NOT in scope:
+
+- Data and functionality that is specific to a single module, and not of general
+  nature.
+- A graphical frontend for interacting with the API.
+
+## Permissions
+
+The `api` module is a public module so that the `login` endpoint can be accessed,
+but otherwise all requests require a token (retrieved from the `login` endpoint)
+or the user accessing the API to be logged in to LORIS.
+
+The following permissions affect accessing data from the API:
+????
```

Comment:
Imaging permissions affect how you can access the imaging data via the API (will be the same for electrophysiology data). Don't know about the rest. I guess there is site and project permission? @xlecours ?

Reply:
That doesn't seem to be the case. I did a grep for "hasPermission" and the only call to the function in the API looks like the data_entry for the candidates endpoint.